src/deploy/osp_deployer/settings/sample_csp_profile.ini

```diff
@@ -171,19 +173,48 @@ StorageBond1Interface1=em2
 StorageBond1Interface2=p2p2
 StorageBondInterfaceOptions=mode=802.3ad miimon=100 xmit_hash_policy=layer3+4 lacp_rate=1
 
+# Note: the following interfaces need to be changed 
+# as per the server model
+# R630 should use 'p2p1', 'p3p1', 'p2p2', 'p3p2'
+# R730 should use 'p4p1', 'p5p1', 'p4p2', 'p5p2'
+# The following lines must be uncommented only if OVS-DPDK is set to enable 
+# See Dell NFV Settings section -> option ovs_dpdk_enable
+ComputeOvsDpdkInterface1=p2p1
+ComputeOvsDpdkInterface2=p3p1
+ComputeOvsDpdkInterface3=p2p2
+ComputeOvsDpdkInterface4=p3p2
+BondInterfaceOvsOptions=bond_mode=balance-tcp lacp=active
+
+
 [Dell NFV Settings]
 #Provide NFV features here.
 
 #Enter value of enable_hpg as True/False for HugePages
-hpg_enable=True
+hpg_enable=true
 #User should give this parameter in same format.
 #Supported values for hpg_size(Size of hugepages) is 2MB and 1 GB.
 #For 2MB (# of hugepages), hpg_num is 49152
 #For 1GB (# of hugepages), hpg_num is 96
 hpg_size=2MB
 
 #Enter value of enable_numa as True/False for NUMA
```

Comment:
Should change "True/False" to "true/false" in the above line for consistency with the xsp profile.